app/controllers/admin/projects_controller.rb

```diff
@@ -5,6 +5,7 @@ class Admin::ProjectsController < ApplicationController
 
   def new
     @project = @facilitator.projects.build
+    @facilitator
```

Comment:
Hey @terencelimsayjian we don't need this here. This is addressed via the `before action`

Reply:
The `before_action :prepare_facilitator` calls the `prepare_facilitator` private method defined below only on the `new` and `create` methods. `prepare_facilitator` assigns `@facilitator`,  so its like an invisible instance variable on the method. 

One can argue that this hides logic, which is confusing- and I agree. On the other hand, Rails adopts this convention since this controller is a nested controller, and only wants to show the code relevant to the main resource of this controller, which are projects.